### DISCUSSION TITLE
Snippets directly selectable on edit tool button

### DISCUSSION THREAD

**Discussion by spc789** [damionx7/Zettel-Notes-Documentation#103] [Feature Requests]
Putting snippets dorectly on the edit toolbar (with a selectable icon)

After a text snippet is inserted in edit mode return the cursor to the edit area and close the insert snippets window and restore the keyboard again based on the visibility of the keyboard before the insert of the snippet was done. (Instead of leaving the insert snippet window open)

So basically if - for example - one edits a snippet with contents 
" --- 
$$yyyyMMdd$$
" 
One can insert that snippet with a single keypress and keep editing afterwards without closing the snippets pane, selecting the keyboard again etc

**Comment by damionx7** [comment]
I have added this feature. It will be live in the next update.
